userspace/libsinsp/event.cpp

```diff
@@ -1402,6 +1404,73 @@ Json::Value sinsp_evt::get_param_as_json(uint32_t id, OUT const char** resolved_
 	return ret;
 }
 
+std::string sinsp_evt::get_cwd(uint32_t id, sinsp_threadinfo *tinfo)
+{
+	std::string cwd = tinfo->get_cwd();
+
+	const ppm_param_info* param_info = &m_info->params[id];
+
+	// Ensure we have the correct parameter type
+	if (param_info->type != PT_FSRELPATH)
+	{
+		ASSERT(param_info->type == PT_FSRELPATH);
```

Comment:
MINOR: If we `ASSERT(param_info->type == PT_FSPATH)` instead (with a comment), we could drop the `if` in line 1683 and route both path types through this function.